ktoml-file/src/commonMain/kotlin/com/akuleshov7/ktoml/file/FileUtils.kt

```diff
@@ -20,6 +20,10 @@ import okio.buffer
 internal fun readAndParseFile(tomlFile: String): List<String> {
     try {
         val tomlPath = tomlFile.toPath()
+        val extension = tomlPath.name.substringAfterLast('.', "")
```

Comment:
I am thinking about making it configurable - may be at least move it under some System Property? :)

Reply:
what do you mean ? making the extension configurable, or the error ?

Reply:
I suppose that an error should be configurable with some System Property